src/plugins/plugin.tooltip.js

```diff
@@ -110,6 +111,7 @@ function splitNewlines(str) {
 
 /**
  * Private helper to create a tooltip item model
+ * @param {Chart} chart
  * @param item - {element, index, datasetIndex} to create the tooltip item for
```

Comment:
This jsdoc was already wrong and is now even more off

Reply:
It looks correct to me. What am I missing?

I did notice that I can add a type for the `item` parameter, if I import from the TypeScript type definitions. (See latest commit.) I'm not sure if you want me to do this or not. (The TypeScript types aren't always identical to what JS uses.)

Reply:
I'm not sure what I saw. I thought there was no `chart` parameter, but not sure why it seemed so. Anyway, looks good to me now.